deploy/clusterrole_binding.yaml

```diff
@@ -1,11 +1,12 @@
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: tower-resource-operator
 subjects:
 - kind: ServiceAccount
   name: tower-resource-operator
+  namespace: tower-operator
```

Comment:
I think I had asked this question somewhere else, but I'm wondering about the implication of having a fixed namespace here.

Reply:
It doesn’t have to be specifically that namespace. The SA can be created in any namespace (usually same namespace as the operator deployment) but this binding will need to be updated to reference that SA’s namespace. Am I right? @xiangjingli

Reply:
@mikeshng  That is true.  In this case,  we specify the SA in the same operator namespace. By this way we want to clearly demonstrate  that the SA is for the operator, not for the k8s runner job.